test/core.jl

```diff
@@ -353,9 +353,45 @@ Context() do ctx
 
     @testset "floating point constants" begin
 
-    t2 = LLVM.DoubleType(ctx)
-    c = ConstantFP(t2, 1.1)
-    @test convert(Float64, c) == 1.1
+    let
+        typ = LLVM.HalfType(ctx)
+        c = ConstantFP(typ, Float16(1.1f0))
+        @test convert(Float16, c) == Float16(1.1f0)
+    end
+    let
+        typ = LLVM.FloatType(ctx)
+        c = ConstantFP(typ, 1.1f0)
```

Comment:
Should use `ctx`.

Reply:
Will change, although I'm not sure why the `ConstantInt` tests above don't do the same.

Reply:
Actually, there isn't a place to provide `ctx` because AIUI the context is taken from the LLVM type. But there is a test a bit below that is missing the context.

Reply:
Issue is here: https://github.com/maleadt/LLVM.jl/blob/27b20eacf63e14e86c434e2b3f29643d52d1b8f2/test/core.jl#L372